client/app/bundles/momentDashboards/components/Chart.jsx

```diff
@@ -0,0 +1,48 @@
+// @flow
```

Comment:
I noticed the file is located in a "bundles" folder. It's okay for now, but we'll file an enhancement to leverage webpack's module bundling feature that will automatically generate bundles for us and load code on demand in the client.

Reply:
yeah it definitely a possibility, though in a side discussion with Julia, we were thinking about keeping React simple on the Rails project and split the UI Lib and other JS libs out into their own Node projects and just `yarn add` them into the project. 

The React integration on Rails deviates quite a lot from `create-react-app` or any other starter to boot, so it'll be a difficult setup for people who want to just work on the UI components as they will now need to learn 2 ecosystems just to get started.

Reply:
That makes sense; in allows for separation of frontend vs backend environments (e.g. local frontend pointing to a QA database). Separation might have to come first before we start optimizing frontend.